website/src/install.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+set -e
+
+# This script installs the Git Town executable in the user's HOME directory.
+
+VERSION=7.6.0           # the version of Git Town to install
```

Comment:
After updating this for each release, is there any other steps for publishing the website?

Reply:
The website auto-updates on each commit to the `main` branch now. See the new setup documented at https://github.com/git-town/git-town/blob/main/documentation/development/website.md#production-environment.